packages/binding/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "target": "es2020",
+    "noEmit": true
+  },
+  "exclude": [
```

Comment:
This should also exclude `build/`. Personally, I'd be more comfortable with an explicit include list rather than using the default include-all behavior